MekHQ/resources/mekhq/resources/CampaignOptionsDialog.properties

```diff
@@ -92,13 +91,76 @@ useSupportEdgeBox.text=Use Edge for Non-Combat Personnel
 useSupportEdgeBox.toolTipText=Allow non-combat personnel to use edge
 useImplantsBox.text=Allow Implants
 useImplantsBox.toolTipText=Allow personnel to have implants (e.g. Manei Domini)
+chkCapturePrisoners.text=Capture Prisoners in Scenarios
+chkCapturePrisoners.toolTipText=If checked, you will have the opportunity to capture prisoners when resolving scenarios
+prisonerStatus.text=Default Prisoner Status:
+prisonerStatus.Prisoner=Prisoner
+prisonerStatus.Bondsman=Bondsman
 altQualityAveragingCheckBox.text=Use higher-precision skill level averaging
 altQualityAveragingCheckBox.toolTipText=Where quality is determined by two skills, attempt to average by number (e.g. 8/4), \nrather than product (e.g. Regular/Green), for more precision.
 useAdvancedMedicalBox.text=Use Advanced Medical Rules (Unofficial)
-useAdvancedMedicalBox.toolTipText=Use the Advanced Medical Rules created by Jayof9s\n(See Advanced_Medical.doc in the docs folder)
+useAdvancedMedicalBox.toolTipText=Use the Advanced Medical Rules created by Jayof9s \n(See Advanced_Medical.doc in the docs folder)
 useDylansRandomXpBox.text=Use Dylan's Random XP (Unofficial)
-useDylansRandomXpBox.toolTipText=Use Dylan's optional random XP on creation of a new person.\n(20% chance each of 0, 1, 2, 3, and randomized between 1 and 8 XP)
-useQuirksBox.text=Use Quirks
+useDylansRandomXpBox.toolTipText=Use Dylan's optional random XP on creation of a new person. \n(20% chance each of 0, 1, 2, 3, and randomized between 1 and 8 XP)
+healWaitingPeriod.text=Days to wait between healing checks by doctors
+naturalHealWaitingPeriod.text=Days to wait for natural healing
+minimumHitsForVees.text=Minimum number of hits for wounded crews and infantry
+useRandomHitsForVees.text=Randomize the number of hits received by crews and infantry
+useRandomHitsForVees.toolTipText=If checked, the number of hits that crews and infantry receive will be randomly selected between the minimum value and 5.
+useTougherHealing.text=Tougher Healing
+useTougherHealing.toolTipText=The healing check will be penalized for every additional hit above 2.
+useTransfers.text=Log Saver - Use Reassign instead of Remove/Assign
+useTimeInService.text=Track Time In Service
+showOriginFaction.text=Show Origin Faction
+randomizeOrigin.text=Randomize Origin
+randomizeDependentsOrigin.text=Randomize Origin for Dependents
+originSearchRadius.text=Random Origin Search Radius
+#family
+FamilyTab.text=Family Options
+minimumMarriageAge.text=Minimum age for marriages:
```

Comment:
None of this appears to be used in this PR.

Reply:
Part of #1533